website/docs/cloud-docs/policy-enforcement/define-policies/prewritten-sentinel.mdx

```diff
@@ -0,0 +1,129 @@
+---
+page_title: Run pre-written Sentinel policies
+description: Learn how to download and install pre-written Sentinel policies created and maintained by HashiCorp.
+---
+
+# Run pre-written Sentinel policies 
+
+This topic describes how to run Sentinel policies created and maintained by HashiCorp. For instructions about how to create your own custom Sentinel policies, refer to [Define custom Sentinel policies](/terraform/cloud-docs/policy-enforcement/define-policies/custom-sentinel).
+
+## Overview
+
+Pre-written Sentinel policy libraries streamline your compliance processes and enhance security across your infrastructure. HashiCorp's ready-to-use policies can help you enforce best practices and security standards across your AWS environment.
+
+Refer to the following resources for details about working with pre-written policies and information about the Sentinel language and framework: 
+
+- [Sentinel documentation](/sentinel/docs) 
+- README documentation included with each of the policy libraries.
+
+Complete the following steps to implement pre-written Sentinel policies in your workspaces:
+
+1. Obtain the policies you want to implement: You download policies directly into your repository or create a fork of the HashiCorp repositories. Alternatively, you can add the Terraform module to your configuration, which acquires the policies and connects them to your workspaces in a single step.
+1. Connect policies to your workspace. If you downloaded policies or forked one or more repositories, you must manually connect the repository containing the policies to your HCP Terraform or Terraform Enterprise workspaces.       
+
+## Requirements
+
+You must use one of the following Terraform applications:
+
+- HCP Terraform
+- Terraform Enterprise v202312-1 or newer
+
+### Permissions
+
+Your HCP Terraform or Terraform Enterprise user account must either be a member of the owners team or have the **Manage Policies** organization-level permissions enabled to create new policy sets and policies. Refer to the following topics for additional information:
+
+- [Organization owners](/terraform/cloud-docs/users-teams-organizations/permissions#organization-owners)
+- [Manage policies](/terraform/cloud-docs/users-teams-organizations/permissions#manage-policies)
+
+### Version control system
+
+You must have a GitHub account connected to HCP Terraform or Terraform Enterprise to manually connect policy sets to your workspaces. Refer to [Connecting VCS Providers](/terraform/cloud-docs/vcs) for instructions.
```

Comment:
Is this GitHub only or will it work with any VCS?

Reply:
GitHub only. I'll add it to the requirements.